org.tinymediamanager.tinyMediaManager.yml

```diff
@@ -0,0 +1,97 @@
+app-id: org.tinymediamanager.tinyMediaManager
+runtime: org.freedesktop.Platform
+runtime-version: "25.08"
+sdk: org.freedesktop.Sdk
+command: tinyMediaManager.sh
+separate-locales: false
+finish-args:
+  # allow access to media files (mounted and local) and settings (configurable by user - per default in ~/.local/share/tinyMediaManager)
+  - --filesystem=/mnt
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=/srv
+  - --filesystem=xdg-data
```

Comment:
There is absolutely not a single reason to request `xdg-data`. Not sure what make you think you should, but it's wrong.